docs/pages/blog/2023-discord-announcement.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'MUI is now on Discord!'
+description: Come join our community to engage in lively discussions, share your projects, and interact with the MUI team
+date: 2023-08-02T00:00:00.000Z
+authors: ['richbustos']
+tags: ['News']
+card: true
+---
+
+<a href="https://mui.com/r/discord"><img src="/static/blog/2023-discord-announcement/discord.png" width="1280" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>
+
+Creating a place where we can interact with our community has been our priority for this year.
+Majority of our discussions and interactions with you have been through our [Twitter](https://twitter.com/MUI_hq), [GitHub](https://github.com/mui), and [marketing page](https://mui.com/).
+
+That's why we have officially launched our [Discord Server](https://mui.com/r/discord/) and would love for you to be part of our community!
+
+Come join to engage in lively discussions, share your projects, and interact with the MUI team & others!
+
+## Why you should join our Discord Server?
+
+There are a plethora of reasons why you should join, but ultimately, it came down to 3 things:
+
+1. **Real-time communication with our community**:
+   As mentioned above, our communication with the community was quite slow.
+   Members would either file an issue on Github, tweet us, or send us a message directly on our website.
+
+   _Discord provides real-time chat capabilities_, allowing you to communicate and collaborate with us and other members, instantly.
+
+   What's a better way to get a real-time response from the MUI team and MUI users?!
+
+2. **Community building**:
+   Want to hear what MUI team is up to?
+   Get the latest update with the products you're using?
+   Want to give us real-time feedback?
```

Comment:
I would cut this line or replace it with a different point since it's just reiterating point 1 on the list.

Reply:
got it!